src/containers/FeedContainer.ts

```diff
@@ -1,29 +1,44 @@
 import { connect } from 'react-redux';
-import { AppState } from '../reducers';
+import { AppState, defaultLocalPosts } from '../reducers';
 import { StateProps, DispatchProps, FeedView } from '../components/FeedView';
 import { AsyncActions, Actions } from '../actions/Actions';
 import { Feed } from '../models/Feed';
 import { getFeedPosts, getYourPosts } from '../selectors/selectors';
 
 export const mapStateToProps = (state: AppState, ownProps: { navigation: any }): StateProps => {
     const feedUrl = ownProps.navigation.state.params.feedUrl;
-    const selectedFeeds = state.feeds.filter(feed => feed != null && feed.feedUrl === feedUrl);
-    // TODO: we should have here state.ownFeeds.find(feed => feed.url === feedUrl) != null,
-    // but author.uri is empty, and the feedUrl from navparams is from post.author.uri (not real feedUrl)
-    // For the same reason feeds/selectedFeeds is empty (errounously)
-    const isOwnFeed = state.author.uri === feedUrl;
+    const feedName = ownProps.navigation.state.params.name;
+
+    const isOwnFeed = feedName === state.author.name;
```

Comment:
we should store the author name in the feed

Reply:
or rather the feed should have an owner

Reply:
This is exactly how it is:
- we store the author of the feed in the `name` property of the feed (along with `url` and `feedUrl` properties)
- we have an `author` property in each post indicating the author of the post

Reply:
By design the idea is that the feedName != authorName

Reply:
That's true that we don't support that on the UI